app/models/instance_user.rb

```diff
@@ -3,4 +3,6 @@ class InstanceUser < ActiveRecord::Base
 
   enum role: { normal: 0, instructor: 1, administrator: 2 }
   belongs_to :user, inverse_of: :instance_users
+
+  scope :ordered_by_username, -> { order('users.name asc') }
```

Comment:
Seem`order` does not support hash expressions when referring another table (in this case `users`) ?